src/config.json

```diff
@@ -909,6 +910,9 @@
     "metapass.fyi"
   ],
   "blacklist": [
+    "ártblocks.net",
+    "ártblocks.io",
+    "ártblocks.com",
```

Comment:
I think `ártblocks.net` should be `xn--rtblocks-7ya.net` in order to handle the tilde.

Reply:
@jakerockland

Reply:
Good catch, thank you!! Fixed!